### ISSUE TITLE
VCS flags var is not being used in VcsExecutive

### ISSUE THREAD

**Issue by chick** [ucb-bar/chisel-testers2#241]
`moreVcsFlags` defined at line 87 in VcsExecutive 
```scala
    val moreVcsFlags = compiledAnnotations.collectFirst { case VcsFlags(flagSeq) => flagSeq }
```
is not being used.
I believe it should be incorporated at line 106 like this
```scala
    val vcsFlags = moreVcsFlags + moreVcsCFlags ++ coverageFlags
```
which would be similar to the usage in VerilatorExecutive.

**Comment by chick** [comment]
@TsaiAnson can you tell by looking if this is broken as I believe it is.

**Comment by TsaiAnson** [comment]
@chick Yeah that doesn't seem to be used. I agree that it should be added to line 106 but with a `++` op:
```scala
val vcsFlags = moreVcsFlags ++ moreVcsCFlags ++ coverageFlags
```

**Comment by chick** [comment]
I believe this is fixed in PR #242
